src/engine.jl

```diff
@@ -40,53 +36,49 @@ type MSession
             # Hide the MATLAB Command Window on Windows
             ccall(engfunc(:engSetVisible ), Cint, (Ptr{Void}, Cint), ep, 0)
         end
-
-        println("A MATLAB session is open successfully")
         new(ep, buf, bufptr)
     end
-
-    MSession() = MSession(default_output_buffer_size)
 end
 
+
 function close(session::MSession)
     # Close a MATLAB Engine session
-
     @assert libeng::Ptr{Void} != C_NULL
-
     r = ccall(engfunc(:engClose), Cint, (Ptr{Void},), session.ptr)
```

Comment:
how is `engClose` supposed to know what  matlab engine to close if you  Null the pointer before the call.

On windows if you don't null it after calling `close` once, calling `close` again, will segfault. (I added tests for this, too)